terminus-ui/input/src/input.component.ts

```diff
@@ -844,6 +844,8 @@ export class TsInputComponent implements
   @Output()
   public readonly selected: EventEmitter<Date> = new EventEmitter();
 
+  // The textual value of the date entered into the input.
+  private textualDateValue = '';
```

Comment:
Let's move this up with the other private properties at the top of this class (should go directly after the `textMaskInputElement` property